### PR TITLE
feat: connection with cloudinary

### PR DESCRIPTION
**Pull request by Dylanyessid**
Se requiere terminar los endpoints de crear y actualizar servicios de formacion (cursos, seminarios, talleres, diplomados)

### CODE REVIEW

File: src/controllers/services.controllers.js [hunk 1]
```diff
@@ -1,16 +1,17 @@
 import { dataSource } from "../../db.js"
+import { uploadFile } from "../services/cloudinary.js"
 
 export const createNewCourse = async(req,res)=>{
     console.log(req.body)
```
Comment (JDMosqueraProgramador) [inline src/controllers/services.controllers.js]: chore: remove console log

File: src/controllers/services.controllers.js [hunk 2]
```diff
@@ -19,8 +20,20 @@ export const createNewCourse = async(req,res)=>{
         inscription_url:"www"
 
     }
+
+    for (const imageField of Object.keys(req.files)){
+        let file = req.files[imageField][0]
+        const res = await uploadFile(file.path)
+        data[imageField] = res.secure_url
+
```
Comment (JDMosqueraProgramador) [inline src/controllers/services.controllers.js]: question: all files will be stored in the same cloud? Also, does the path contain a single name or you will organize it by folders?

File: src/services/cloudinary.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import {v2 as cloudinary,} from "cloudinary"
+import  dotenv  from 'dotenv';
+
+dotenv.config()
```
Comment (JDMosqueraProgramador) [inline src/services/cloudinary.js]: suggestion: I think you can call this only once in the main file, or build a configService to centralize all env variables

**Review by JDMosqueraProgramador — COMMENTED** [review]
I suggest to use consistent ";" in all statements

File: src/controllers/services.controllers.js (hunk 2 above)
Comment (Dylanyessid) [reply src/controllers/services.controllers.js]: All files will stored in cloudinary, in a folder called "pyglo-app"

**Comment by Dylanyessid** [comment]
Closed to add corrections

**Pull request closed without merging**